semaphore/integration_tests.py

```diff
@@ -0,0 +1,105 @@
+#/usr/bin/env python
+from datetime import date, time, datetime, timedelta
+from random import randrange, getrandbits
+import json
+import requests
+
+SPLUNK_HOST = "https://mbta.splunkcloud.com:8089"
+SAVED_SEARCH = "Top 10 trip plans %2B 10 latest trip plans"
+OTP_DEV = "https://dev.otp.mbtace.com"
+OTP_PROD = "https://prod.otp.mbtace.com"
+
+
+def splunk_json_request(path):
+    return requests.get(f"{SPLUNK_HOST}{path}",
+                        params={"output_mode": "json"},
+                        auth=("lboyarsky@mbta.com", "XXX"),
```

Comment:
Should use tokens instead of credentials, blocked on https://app.asana.com/0/810933294009540/1131320492706584/f